### ISSUE TITLE
Layers[152].output and Layers[92].output

### ISSUE THREAD

**Issue by smalldroid** [qqwweee/keras-yolo3#483]
Why layer index is 152 and 92? Does Layers of Model include Conv2d, BatchNorm and LeakyRelu? Thanks!

**Comment by i-chaochen** [comment]
Same confuse here, have you figured it out?  @smalldroid

**Comment by datduonguva** [comment]
If you list all the layers out, you will see why. Basically, those are the two output layers of darknet model (for tiny yolo). For full yolo, the are 3 outputs of darknet model. Please refer to the paper for more explanation.
